XenModel/Messages.resx

```diff
@@ -12968,20 +12968,20 @@ Please check your settings and try again.</value>
     <value>{0}: Check skipped because WLB is enabled on pool {1}.</value>
   </data>
   <data name="UPDATE_DISMISS_ALL_CONTINUE" xml:space="preserve">
-    <value>You have applied filters to the list of updates. Do you want to dismiss all updates from every connected server, or only the updates you have chosen to view? In both cases the dismissed updates will be removed from the servers permanently.
+    <value>You have applied filters to the list of updates. Do you want to dismiss all updates in {0} from every connected server, or only the updates you have chosen to view?

```

Comment:
I think we can keep the messages simple and avoid the client placeholder and string formatting by using the term "update notifications" instead of "updates". I.e. `You have applied filters to the list of update notifications. Do you want to dismiss all update notifications for every connected server, or only the update notifications you have chosen to view?` Then it's clear that we only touch the alerts and not the actual updates. Same in the two messages below.